source/differences.rst

```diff
@@ -38,6 +38,7 @@ New data elements
    - ``browserJavascriptEnabled``
    - ``whiteListStatus``
    - ``whiteListStatusSource``
+   - ``payTokenSource``
```

Comment:
This field is not one we accept right now, so it should be removed from here.

Reply:
But it's included in the specs => https://docs.3dsecure.io/3dsv2/specification_220.html#attr-AReq-payTokenSource

Reply:
Aah, I see that :) I remembered wrongly. That's for double checking :+1: